ApplicationLogs/ApplicationLogs.csproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo" Version="2.9.4" />
+    <PackageReference Include="Neo" Version="2.10.1-CI00001" />
```

Comment:
Should waiting for the release of neo core.

Reply:
That is fine, this will just stay here for reference for people building master that need this plugin to work, right?

Reply:
Hi @erikzhang,
    Do you think that we should create another branch, for example, "plugins dev" and merge this and other upcoming changes there?
    In this sense we could aggregate everything that is needed for the next release.